src/linker/Linker/Annotations.cs

```diff
@@ -630,23 +631,27 @@ internal bool DoesMemberRequireUnreferencedCode (IMemberDefinition member, [NotN
 			};
 		}
 
-		internal bool ShouldSuppressAnalysisWarningsForRequiresUnreferencedCode (ICustomAttributeProvider? originMember)
+		internal bool ShouldSuppressAnalysisWarningsForRequiresUnreferencedCode (ICustomAttributeProvider? originMember, [NotNullWhen (true)] out RequiresUnreferencedCodeAttribute? attribute)
```

Comment:
Now that we return an attribute in this method I think is the same as DoesMemberRequireUnreferencedCode, and given that you changed the calls to DoesMemberRequireUnreferencedCode to ShouldSuppressAnalysisWarningsForRequiresUnreferencedCode I think now nothing calls DoesMemberRequireUnreferencedCode, DoesMethodRequireUnreferencedCode, and DoesFieldRequireUnreferencedCode. I'm fine with 'ShouldSuppressAnalysisWarningsForRequiresUnreferencedCode' method is more readable and correct but probably we should get rid of unused methods